### PR TITLE
Hypervisor cluster profile for api800/1000/1200

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Added resources Hypervisor cluster profile for api800/1000/1200

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [y] All tests pass (`$ rake test`).
  - [y] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [y] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [y] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ## 3.2.1(unreleased)
```
Comment (soodpr) [inline CHANGELOG.md]: the new version should be 3.3.0 since its a new resource added to it as per semantic versioning

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+### Notes
+- This release adds support to Oneview Rest API version 800, 1000, and 1200 for C7000 and Synergy to the features Hypervisor Cluster Profile.
+
+### New features and resources
+  - Added support for following resources
```
Comment (soodpr) [inline CHANGELOG.md]: remove this line

File: libraries/matchers.rb [hunk 3]
```diff
@@ -31,6 +31,7 @@
     oneview_fc_network:                 standard_actions + scope_actions + %i[reset_connection_template],
     oneview_fcoe_network:               standard_actions + scope_actions + %i[reset_connection_template],
     oneview_firmware:                   %i[add remove create_custom_spp],
+    oneview_hypervisor_cluster_profile: standard_actions + %i[create_or_update],
```
Comment (soodpr) [inline libraries/matchers.rb]: i think it should be update here as per the actions defined

**Review by soodpr — CHANGES_REQUESTED** [review]
I have reveiwed your code, please incorporate the changes and let me know once its done

File: libraries/matchers.rb (hunk 3 above)
Comment (AsisBagga) [reply libraries/matchers.rb]: @soodpr Corrected

File: CHANGELOG.md (hunk 2 above)
Comment (AsisBagga) [reply CHANGELOG.md]: @soodpr Corrected

File: CHANGELOG.md (hunk 1 above)
Comment (AsisBagga) [reply CHANGELOG.md]: @soodpr Corrected

**Review by soodpr — APPROVED** [review]
LGTM

File: examples/hypervisor_cluster_profile.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+# NOTE:  Hypervisor Cluster Profile resource works for client version  API800 and greater.
+
+my_client = {
+  url: ENV['ONEVIEWSDK_URL'],
+  user: ENV['ONEVIEWSDK_USER'],
+  password: ENV['ONEVIEWSDK_PASSWORD'],
+  api_version: 800
```
Comment (AnushaY1916) [inline examples/hypervisor_cluster_profile.rb]: It would be good if we put the latest supported version.

**Review by AnushaY1916 — APPROVED** [review]

File: examples/hypervisor_cluster_profile.rb (hunk 4 above)
Comment (AsisBagga) [reply examples/hypervisor_cluster_profile.rb]: @AnushaY1916 Updated

**Pull request merged**